ai_serving/template/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:20.04

```

Comment:
Can we try switching this to python:3-slim, thats what jaseci is based on these days and would save image size for public dockerhub. Give it a try , if too tricky we can keep ubuntu.